__tests__/app.test.js

```diff
@@ -304,7 +304,20 @@ describe('DELETE /api/comments/:comment_id', () => {
     })
 })
 
-
+describe('GET /api/users',()=> {
+    test('GET 200 responds with an array of users each with username, name and avatar_url properties', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+            body.users.forEach((user) => {
+                expect(user).toHaveProperty('username')
```

Comment:
you might want to add a second argument to each of these, `toHaveProperty` s, you can use `expect.any(String)` to validate that they are strings